README.md

```diff
@@ -1,246 +1,187 @@
 
-`$ pipenv install`
+* Can either return the underlying env reward or the reward given by the reward predictor, which is loaded and updated throughout the 
+course of reward predictor training 
 
-However, note that TensorFlow must be installed manually. Either:
+* Can be used with any gym-compatible RL algorithm, which can be passed the reward-predictor reward without needing to be specially modified
 
-`$ pipenv run pip install tensorflow`
 
-or
 
-`$ pipenv run pip install tensorflow-gpu`
+## Setup
 
-depending on whether you have a GPU. (If you run into problems, try installing
-TensorFlow 1.6.0, which was used for development.)
+This repo is structured as a Python package, and can be installed by running 
 
-If you want to run tests, also run:
+`python setup.py install`
 
-`$ pipenv install --dev`
+Tests can be run with
 
-Finally, before running any of the scripts, enter the environment with:
+`pytest tests/`
 
-`$ pipenv shell`
-
-### Running
-
-All training is done using [`run.py`](drlhp/deprecated/run.py). Basic usage is:
-
-`$ python3 run.py <mode> <environment>`
-
-Supported environments are
-[`MovingDotNoFrameskip-v0`](https://github.com/mrahtz/gym-moving-dot),
-`PongNoFrameskip-v4`, and `EnduroNoFrameskip-v4`.
-
-### Training with original rewards
-
-To train using the original rewards from the environment rather than rewards
-based on preferences, use the `train_policy_with_original_rewards` mode.
-
-For example, to train Pong:
-
-`$ python3 run.py train_policy_with_original_rewards PongNoFrameskip-v4 --n_envs 16 --million_timesteps 10`
-
-### Training end-to-end with preferences
-
-Use the `train_policy_with_preferences` mode.
-
-For example, to train `MovingDotNoFrameskip-v0` using *synthetic* preferences:
-
-`$ python3 run.py train_policy_with_preferences MovingDotNoFrameskip-v0 --synthetic_prefs --ent_coef 0.02 --million_timesteps 0.15`
-
-On a machine with a GPU, this takes about an hour. TensorBoard logs (created in
-a new directory in `runs/` automatically) should look something like:
-
-![](drlhp/deprecated/images/moving-dot-graphs.png)
-
-To train Pong using *synthetic* preferences:
-
-`$ python3 run.py train_policy_with_preferences PongNoFrameskip-v4 --synthetic_prefs --dropout 0.5 --n_envs 16 --million_timesteps 20`
-
-On a 16-core machine without GPU, this takes about 13 hours. TensorBoard logs
-should look something like:
-
-![](drlhp/deprecated/images/pong-graphs.png)
-
-To train Enduro (a modified version with a time limit so the weather doesn't change, which the paper notes can confuse the reward predictor) using *human* preferences:
-
-`$ python3 run.py train_policy_with_preferences EnduroNoFrameskip-v4 --n_envs 16 --render_episodes`
-
-You'll see two windows: a larger one showing a pair of examples of agent
+## Human Preferences GUI
+When not training with human rather than synthetic preferences, you'll see two windows: a larger one showing a pair of examples of agent
```

Comment:
```suggestion
When training with human rather than synthetic preferences, you'll see two windows: a larger one showing a pair of examples of agent
```

Reply:
Heh, "not" is such an unfortunate word to have as a typo, when it comes to preservation of semantic meaning :P